kolibri/plugins/learn/assets/src/views/LearnIndex.vue

```diff
@@ -288,6 +262,23 @@
       userPluginUrl() {
         return urls['kolibri:kolibri.plugins.user:user'];
       },
+      learnBackPageRoute() {
```

Comment:
I'm not sure if this is the code that is relevant, but when I go to content from the Recommended page and click back (the button or the browser button) I am taken to that content's root in its channel when I ought to be returned to Recommended.

Reply:
Ah - thanks for noticing that. Will update!